airflow/providers/openlineage/extractors/base.py

```diff
@@ -28,14 +28,18 @@
     warnings.simplefilter("ignore", DeprecationWarning)
     from openlineage.client.facet import BaseFacet as BaseFacet_V1
 from openlineage.client.facet_v2 import JobFacet, RunFacet
+from packaging.version import Version
 
+from airflow import __version__ as AIRFLOW_VERSION
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.state import TaskInstanceState
 
 # this is not to break static checks compatibility with v1 OpenLineage facet classes
 DatasetSubclass = TypeVar("DatasetSubclass", bound=OLDataset)
 BaseFacetSubclass = TypeVar("BaseFacetSubclass", bound=Union[BaseFacet_V1, RunFacet, JobFacet])
 
+IS_AIRFLOW_2_10_OR_HIGHER = Version(Version(AIRFLOW_VERSION).base_version) >= Version("2.10.0")
```

Comment:
We have that in the utils. you can just import it from there
https://github.com/apache/airflow/blob/10f250347d17e8f1362e8fffc3dddce655a11cf7/airflow/providers/openlineage/utils/utils.py#L67

Reply:
I adjusted all the usage within the provider to import this one from utils, thanks for the catch.